pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -4107,24 +4107,27 @@ void FullscreenUI::DrawControllerSettingsPage()
 							bsi->DeleteValue(section, key.c_str());
 						else
 							bsi->SetStringValue(section, key.c_str(), binds_string.c_str());
+
+						SetSettingsChanged(bsi);
 					});
 			}
 
-			const TinyString freq_key = TinyString::from_fmt(FSUI_FSTR("Macro {} Frequency"), macro_index + 1);
+			const TinyString freq_key = TinyString::from_fmt("Macro{}Frequency", macro_index + 1);
+			const TinyString freq_label = TinyString::from_fmt(FSUI_FSTR("Macro {} Frequency"), macro_index + 1);
 			s32 frequency = bsi->GetIntValue(section, freq_key.c_str(), 0);
 			const SmallString freq_summary =
 				((frequency == 0) ? TinyString(FSUI_VSTR("Macro will not auto-toggle.")) :
 									TinyString::from_fmt(FSUI_FSTR("Macro will toggle every {} frames."), frequency));
 			if (MenuButton(FSUI_ICONSTR(ICON_FA_LIGHTBULB, "Frequency"), freq_summary.c_str()))
-				ImGui::OpenPopup(freq_key.c_str());
+				ImGui::OpenPopup(freq_label.c_str());
 
 			const std::string pressure_key(fmt::format("Macro{}Pressure", macro_index + 1));
```

Comment:
These should probably be TinyStrings as well, to avoid the heap allocations, but that's not a bug.